lib/Test/WWW/Stub.pm

```diff
@@ -59,19 +48,14 @@ sub import {
 sub register {
     my ($class, $uri_or_re, $app_or_res) = @_;
     $app_or_res //= [200, [], []];
-    my $old_handler = $Handlers->{$uri_or_re};
+    my $old_handler = $Intercepter->get($uri_or_re);
```

Comment:
This method name seems not to be obvious to return a handler.
How about renaming this method to something like `get_handler` ?